rules/o365azure/azuread_highfail_mfa_challenges.yml

```diff
@@ -0,0 +1,21 @@
+title: Users with a high count of failed MFA challenges
+id: 337ff30b-97a1-4d92-9bc8-fbc1a618d20a
+status: experimental
+description: detect accounts for which the password was compromised but the authentication failed because MFA was enabled
+references:
+  - https://0x00sec.org/t/a-blue-team-guide-to-azure-office-365-monitoring/14411
+author: 'juju4 (rule), 0x00sec.org (idea)'
+date: 2019/06/06
+tags:
+    - attack.t1078
+logsource:
+    product: o365
+falsepositives:
+    - Penetration Test
+    - Buggy products
+    - Unknown
+level: medium
+detection:
+    selection:
+        LogonError: UserStrongAuthClientAuthNRequiredInterrupt
+    condition: selection | count(user) by clientip > 10
```

Comment:
@thomaspatzke that's splunk syntax, isn't it? We do not support that yet, right?